src/services/guided-tour.service.ts

```diff
@@ -0,0 +1,141 @@
+import * as angular from 'angular';
+import * as hopscotch from 'hopscotch';
+import * as $ from 'jquery';
+import * as _ from 'lodash';
+
+interface IGuidedTourService {
+  startTour(tourConfig: any);
+  cancelTour();
+}
+
+export class GuidedTourService implements IGuidedTourService {
```

Comment:
We might want to eventually push this into common so that we can reuse it for other console pages.